conf/messages.en

```diff
@@ -47,3 +47,15 @@ signedOut.guidance = We did not save your answers.
 
 unauthorised.title = You can’t access this service with this account
 unauthorised.heading = You can’t access this service with this account
+
+inactivityRequest.title = Do you want to submit an inactivity request?
+inactivityRequest.heading = Do you want to submit an inactivity request?
+inactivityRequest.option1 = Yes
+inactivityRequest.option2 = No
+inactivityRequest.checkYourAnswersLabel = Do you want to submit an inactivity request?
+inactivityRequest.error.required = Select yes if you want to submit an inactivity request
+inactivityRequest.change.hidden = do you want to submit an inactivity request
+inactivityRequest.p1 = You can submit an inactivity request to tell HM Revenue & Customs (HMRC) that you do not anticipate paying subcontractors in the next six months.
+inactivityRequest.p2 = If this request is accepted, you won''t need to send us a monthly return.
```

Comment:
please use:
inactivityRequest.p2 = If this request is accepted, you won’t need to send us a monthly return.